docs/content/SegmentedControl.mdx

```diff
@@ -0,0 +1,190 @@
+---
+title: SegmentedControl
+status: Draft
+description: Used to select an option from a short list and immediately apply the selection
+---
+
+## Examples
+
+### Simple
+
+```jsx live drafts
+<SegmentedControl aria-label="File view">
+  <SegmentedControl.Item>Preview</SegmentedControl.Item>
+  <SegmentedControl.Item>Raw</SegmentedControl.Item>
+  <SegmentedControl.Item>Blame</SegmentedControl.Item>
+</SegmentedControl>
+```
+
+### With icons and labels
+
+```jsx live drafts
+<SegmentedControl aria-label="File view">
+  <SegmentedControl.Item leadingIcon={EyeIcon}>Preview</SegmentedControl.Item>
+  <SegmentedControl.Item leadingIcon={FileCodeIcon}>Raw</SegmentedControl.Item>
+  <SegmentedControl.Item leadingIcon={PeopleIcon}>Blame</SegmentedControl.Item>
+</SegmentedControl>
+```
+
+### With icons only
+
+```jsx live drafts
+<SegmentedControl aria-label="File view">
+  <SegmentedControl.IconItem icon={EyeIcon} aria-label="Preview" />
+  <SegmentedControl.IconItem icon={FileCodeIcon} aria-label="Raw" />
+  <SegmentedControl.IconItem icon={PeopleIcon} aria-label="Blame" />
+</SegmentedControl>
+```
```

Comment:
Alternate idea:
```jsx
<SegmentedControl aria-label="File view" hideLabels>
  <SegmentedControl.Item leadingIcon={EyeIcon}>Preview</SegmentedControl.Item>
  <SegmentedControl.Item leadingIcon={FileCodeIcon}>Raw</SegmentedControl.Item>
  <SegmentedControl.Item leadingIcon={PeopleIcon}>Blame</SegmentedControl.Item>
</SegmentedControl>
```

Reply:
How do you feel about using aria-label instead of `hideLabels`

```jsx
<SegmentedControl aria-label="File view">
  <SegmentedControl.Item leadingIcon={EyeIcon} aria-label="Preview" />
  <SegmentedControl.Item leadingIcon={FileCodeIcon} aria-label="Raw" />
  <SegmentedControl.Item leadingIcon={PeopleIcon} aria-label="Blame" />
</SegmentedControl>
```

Reply:
I'm good with
```
<SegmentedControl aria-label="File view">
  <SegmentedControl.IconButton icon={EyeIcon} aria-label="Preview" />
  <SegmentedControl.IconButton icon={FileCodeIcon} aria-label="Raw" />
  <SegmentedControl.IconButton icon={PeopleIcon} aria-label="Blame" />
</SegmentedControl>
```

I think the only difference between that and what I have now is the name `IconButton` instead of `Item`

Reply:
I'm good with
```
<SegmentedControl aria-label="File view">
  <SegmentedControl.IconButton icon={EyeIcon} aria-label="Preview" />
  <SegmentedControl.IconButton icon={FileCodeIcon} aria-label="Raw" />
  <SegmentedControl.IconButton icon={PeopleIcon} aria-label="Blame" />
</SegmentedControl>
```

I think the only difference between that and what I have now is the name `IconButton` instead of `Item`